Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -613,6 +613,10 @@ def audit_homepage
       problem "The homepage should start with http or https (URL is #{homepage})."
     end
 
+    unless homepage =~ %r{^(.+)://(.*)/(.*)}
```

Comment:
It's was the missing negation that's causing problems, but the basic case don't have extra characters after the slash, so it fails to match.
Ah okay, the edit version should work.